apInitPassthru.py

```diff
@@ -61,10 +61,9 @@ def find_ardupilot_serial_ports():
     '''
     apportList = []
     for port in portList:
-        if port.vid == 0x1209 and port.pid == 0x5740:
-            if 'ardupilot' in port.description.lower() and 'mavlink' in port.description.lower():
-                if 'ardupilot' in port.manufacturer.lower():
-                    apportList.append(port.device)
+        if port.vid == 0x1209 and (port.pid == 0x5740 or port.pid == 0x5741):
+            if 'ardupilot' in port.manufacturer.lower():
```

Comment:
are these changes a win vs lin thing? Or for another reason?
I mean, I would not have expected that ArduPilot sets these differently dependening on which os is used...

Reply:
No, at least part of this would also be required on Windows for my boards.  Some flight controller boards including my FlywooF405S-AIO and Pixhawk1 clone use this pid.  Google tells me that the MambaH743v4 also uses this pid.  At least on Linux, all of these boards also have their board name in description instead of the generic description like "Ardupilot MAVLink".  Is port.description even part of the USB device descriptor?  I think it was intended only as a human readable string I haven't checked if it is the same on Linux and Windows.